blazor/pager/behaviour-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Behaviour Settings in Blazor Pager Component | Syncfusion
+description: Checkout here and learn about all the basic functionalities of rendering the Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Behaviour Settings in Pager Component
```

Comment:
Relation ship between API and priorities are not clearly explained.
Realtime use case queries are not covered